src/main/java/com/locallampoon/fiveh/ui/PanelStyles.java

```diff
@@ -0,0 +1,21 @@
+package com.locallampoon.fiveh.ui;
+
+import java.awt.*;
+
+class PanelStyles {
```

Comment:
Weren't you talking about trying to do this in an Enum?

Reply:
Just from the top of my head either Enum or constant were my possible appraches. I chose constant so that it can be all on 1 line as opposed to creating an Enum constructor and assigning values for each style.

Reply:
Seems good for now.